packages/server/src/modules/app/tests/request.ts

```diff
@@ -0,0 +1,78 @@
+import supertest, { SuperTest, Test } from 'supertest';
+
+import application from '@application';
+import { TestRequest } from '@types';
+
+const tokenCookie =
```

Comment:
This is not what we want. Cookie should be set individually in each `beforeEach`.

Come up with suggestion for that. You have to combine seeded user and their token.

Reply:
I wrote in the description that I still have to come up with this part.

For now it is set each time, except if `noSession` is called